### ISSUE TITLE
Cryosting is quite useless currently.

### ISSUE THREAD

**Issue by SeikoTheSeal** [tgstation/-tg-station#8276]
Since #7790 , cryosting became a waste of points. Sting an officer in default gear, 17 ticks later, officer starts taking amazing 0.5 burn per tick and suffers minor slowdown (the cyan "Too cold" icon) and nutrition drain. 30u(one sting) of frost oil equals to 54,5 burn after 75 ticks. You are better off throwing pens at them. Using it against people is space suits is tad better, but still so painfully slooooow.

Combat wise, it has litteraly zero uses, slowdown kicks in too slow, damage is meaningless. Trying to assassinate someone with just it, target will have time to run to nearest vending machine/medbay ten times before they even take any serious damage.

TL:DR

Cryosting is terribly weak since ice was removed.

**Comment by phil235** [comment]
Actually the problem comes from the metabolization speed being changed to 3ticks for all reagents instead of just new goon chems. It will be fixed in #8058 , this should change the speed and damage of cryosting back to original levels.

**Comment by xxalpha** [comment]
I'd say the problem is that one PR that nerfs Cryosting.

**Comment by SeikoTheSeal** [comment]
Alright, thanks for explaining Phil, this can be closed then i guess.
